operator/roles/ansible-service-broker/tasks/trusted_ca_bundle.yml

```diff
@@ -0,0 +1,69 @@
+---
+
+- name: check if trusted CA configmap exists
```

Comment:
For consistency, I'd start the name with a capital letter, `Check`

Reply:
:+1: